package.json

```diff
@@ -22,7 +22,7 @@
     "typescript": "^4.1.5"
   },
   "dependencies": {
-    "@o1labs/client-sdk": "^0.2.5",
+    "@o1labs/client-sdk": "^1.0.0",
```

Comment:
Need to add support for both libraries and separate devnets from mainnet. dev/test nets need to be pinned at 0.2.5, only mainnet runs on 1.0.0.